discoverApi.ts

```diff
@@ -0,0 +1,47 @@
+import axios, { AxiosResponse } from 'axios';
+// require('dotenv').config();
+import 'dotenv/config'
+const apiKey = process.env.API_KEY;
+
+
+interface MovieData {
+    results: any;
+  adult: boolean;
+  backdrop_path: string;
+  genre_ids: number[];
+  id: number;
+  original_language: string;
+  original_title: string;
+  overview: string;
+  popularity: number;
+  poster_path: string;
+  release_date: string;
+  title: string;
+  video: boolean;
+  vote_average: number;
+  vote_count: number;
+}
+
+async function fetchData(): Promise<AxiosResponse<MovieData>> {
+  const url =
+    'https://api.themoviedb.org/3/discover/movie?include_adult=false&include_video=false&language=en-US&page=1&sort_by=popularity.desc';
```

Comment:
Are you always going to send requests to the base URL with the same query strings?